Modules/_datetimemodule.c

```diff
@@ -6581,7 +6600,7 @@ _datetime_exec(PyObject *module)
     }
 
     PyDateTime_TimeZone_UTC = x;
-    CAPI.TimeZone_UTC = PyDateTime_TimeZone_UTC;
+    CAPI->TimeZone_UTC = PyDateTime_TimeZone_UTC;
```

Comment:
I suggest to move this assignement inside get_datetime_capi(), and only call get_datetime_capi() after PyDateTime_TimeZone_UTC is defined.

Minor suggestion, please mention that the reference is borrowed:

```suggestion
    CAPI->TimeZone_UTC = PyDateTime_TimeZone_UTC; // borrowed ref
```

Reply:
No problem. And I add a assert to make sure `get_datetime_capi()` is called after PyDateTime_TimeZone_UTC has been initialized.